src/pages/discover/profiles.tsx

```diff
@@ -62,6 +62,7 @@ export default function DiscoverProfilesTab(): JSX.Element {
   return (
     <CardGridWithHeader<ProfilePreviewProps>
       cardContext={{
+        cardType: 'PROFILES', // Make ENUM
```

Comment:
Since we only have a limited number of card types I don't think it's nescecarily a bad idea to have the grid have some knowledge of the card type, but I defer to you and added a bigGridClassNameOverride prop instead.